fuzzers/frida_libpng/README.md

```diff
@@ -27,4 +27,48 @@ This means running --cores each client will start itself again to listen for cra
 By restarting the actual fuzzer, it can recover from these exit conditions.
 
 After building the libpng-harness, too, you can run `find . -name libpng-harness.so` to find the location of your harness, then run
-`./target/release/frida_libpng ./libpng-harness.so LLVMFuzzerTestOneInput ./libpng-harness.so --cores=0`
\ No newline at end of file
+`./target/release/frida_libpng ./libpng-harness.so LLVMFuzzerTestOneInput ./libpng-harness.so --cores=0`
+
+## Windows
+You can also fuzz libpng-1.6.37 on windows with frida mode!
+(but you need to grab msys2 for compiling libpng)
```

Comment:
Can't we even fuzz a binary version of libpng?

Reply:
it can.
idk if this answers the question, but actually we don't need to wget and tar and build libpng manually.
After installing this package (https://packages.msys2.org/package/mingw-w64-x86_64-libpng) then libpng16.a resides in /mingw64/lib/libpng16.a
then
```
g++ -O3 harness.o /mingw64/lib/libpng16.a -shared -lz -o libpng-harness.dll
./frida_libpng.exe ./libpng-harness.dll LLVMFuzzerTestOneInput ./libpng-harness.dll --cores=0
```
and you can fuzz libpng

Reply:
What if we use the normal libpng from here http://gnuwin32.sourceforge.net/packages/libpng.htm then we don't need msys2, I guess?

Reply:
I looked at that but I could not compile it into a libpng-harness.dll with harness.o.
probably it's too old
plus it's 32bit code

Reply:
ehm, something's odd with windows's clang.
The symbols (like LLVMFuzzerTestOneInput) got disappeared after linking into a dll

Reply:
Are u using __declspec(dllexport) ?

Reply:
didn't know that.
Now it works like a magic 🤩